code/API_definitions/Traffic_Influence.yaml

```diff
@@ -135,32 +135,40 @@ info:
     "trafficInfluenceID"). All the created resources are aggregated by the
     Application (identified by "appId").\
     \
-    **appId**
+    **appId:**
```

Comment:
I saw that some other parts could get this as well.
Something like `sed 's/\([A-Za-z0-9]\)\*\*/\1\:\*\*/g` should do the trick.